app/core/Multichain/test/utils.test.ts

```diff
@@ -87,6 +91,22 @@ const mockBTCAccount: InternalAccount = {
   type: BtcAccountType.P2wpkh,
 };
 
+const mockSolAccount: InternalAccount = {
+  address: SOL_ADDRESSES,
+  id: '1',
+  type: SolAccountType.DataAccount,
+  methods: [SolMethod.SendAndConfirmTransaction],
+  options: {},
+  metadata: {
+    name: 'Solana Account',
+    importTime: 1684232000456,
+    keyring: {
+      type: KeyringTypes.hd,
+    },
+  },
+  scopes: [SolScopes.Namespace],
```

Comment:
On the extension side, we usually use a full list of scopes with only CAIP-2 chain IDs here. We're about to enforce this for some account types, so would be great to fix it now:

```suggestion
  scopes: [SolScopes.Mainnet, SolScopes.Testnet, SolScopes.Devnet],
```